lib/jira/client/index.js

```diff
@@ -40,19 +40,14 @@ const withinIssueKeyLimit = (resources) => {
  */
 module.exports = async (id, installationId, jiraHost) => {
   const { jiraHost: baseURL, sharedSecret: secret } = await Installation.getForHost(jiraHost)
-
   const instance = getAxiosInstance(id, installationId, baseURL, secret)
 
   const client = {
     baseURL: instance.defaults.baseURL,
     issues: {
-      // eslint-disable-next-line camelcase
-      get: (issue_id, query = { fields: 'summary' }) => instance.get(`/rest/api/latest/issue/:issue_id`, {
-        fields: {
-          ...query,
-          issue_id
-        }
-      }),
+      get: (issueId, params = { fields: 'summary' }) => {
```

Comment:
What is the motivation for renaming `query` to `params` and should we also rename the argument in `getAll`?